src/server/managers/AppListenerManager.ts

```diff
@@ -1196,4 +1203,58 @@ export class AppListenerManager {
 
         return descriptor;
     }
+
+    private async executePostUserCreated(data: IUserContext): Promise<void> {
+        const context = Object.freeze(data);
```

Comment:
`Object.freeze` performs a shallow freeze only, in these cases it's better to use `Utilities.deepFreeze` to ensure inner objects are also frozen